decidim-core/app/packs/stylesheets/decidim/utils/_settings.scss

```diff
@@ -880,4 +880,4 @@ $grid-container-padding: $grid-padding-gutters;
 $grid-container-max: $global-width;
 $xy-block-grid-max: 8;
 
-@import "decidim/decidim-settings";
+@import "../decidim-settings";
```

Comment:
The idea of this file is to overrule the default Decidim SCSS variables within the instance.

I'm wondering how easy it is after this to overrule these configurations from the instance itself? Do we need to create a whole custom "pack" within the instance and add everything we need there?

This same applies for the JS packs which are already merged to the core. Right now we can override single Decidim JS files just by adding the file to the same path in the assets directory as in the core.

I would hope this aspect would be also considered when structuring the SCSS and JS files, so that it would be as easy as possible to add overrides to them.

I understand webpacker works in quite a different way than the assets pipeline but right now, the structure of the files does not seem to make it easy to overrule the core defaults (e.g. Foundation SCSS variables).